cmd/bisync/deltas.go

```diff
@@ -264,6 +264,7 @@ func (b *bisyncRun) applyDeltas(ctx context.Context, ds1, ds2 *deltaSet) (change
 	// check is a read-only operation by its nature, so it's already "dry" in that sense.
 	ctxNew, ciCheck := fs.AddConfig(ctx)
 	ciCheck.DryRun = false
+	ciCheck.NoUpdateModTime = true
```

Comment:
Please write why in the commit message!  (I can't comment on the commit messages)

Reply:
This is made moot by a different change, so I removed it.